sycl/source/detail/queue_impl.hpp

```diff
@@ -244,7 +241,7 @@ class queue_impl {
     if (MPropList.has_property<property::queue::enable_profiling>()) {
       CreationFlags |= PI_QUEUE_PROFILING_ENABLE;
     }
-    RT::PiQueue Queue;
+    RT::PiQueue Queue{};
```

Comment:
Why do you need `{}` here?

Reply:
To initialize the variable.

Reply:
> To initialize the variable.

With what? In other places there is an explicit initialization to `nullptr`.

Reply:
Trying to be more C++ developer. Strictly speaking I do not know that PiQueue is and alias of a pointer to some time. 
With zero if PiQueue is a pointer. https://en.cppreference.com/w/cpp/language/value_initialization